lib/node-writable.js

```diff
@@ -0,0 +1,21 @@
+var Writable = require('stream').Writable,
```

Comment:
We're not running any of our tests on Node 0.8 so I guess by virtue of that we don't.